key.core/src/main/java/de/uka/ilkd/key/java/Services.java

```diff
@@ -327,6 +327,14 @@ public Counter getCounter(String name) {
         return c;
     }
 
+    /**
+     * Reset all counters associated with this service.
+     * Only use this method if the proof is empty!
```

Comment:
Do you think it would be useful to add a guard? I think this is dangerous.

Reply:
I have added a guard.